src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -166,6 +166,426 @@ E. Click the *Tasks* tab, select an *Preview_Results* task and click on the *Pre
+
+One of the most important notions in workflows is the dependencies between tasks. To specify this information, use the
+`dep` parameter. Its value should be a list of all tasks on which the new task depends. Example:
+
+```python
+#%task(name=myTask,dep=[parentTask1,parentTask2])
+print('Hello world')
+```
+
+===== Generic information
+
+To specify the values of some advanced ProActive variables called
+https://doc.activeeon.com/latest/user/ProActiveUserGuide.html#_generic_information[Generic Information], you should the
+provide the `generic_info` parameter. Its value should be a list of tuples `(key,value)` that corresponds the names and
+adequate values of the Generic Information. Example:
+
+```python
+#%task(name=myTask, generic_info=[(var1,value1),(var2,value2)])
+print('Hello world')
+```
+
+===== Export/import variables
+
+The `export` and `import` parameters make variables propagation possible between the different tasks of a workflow.
+If `myTask1` variables `var1` and `var2` are needed in `myTask2`, both pragmas have to specify this information as follow:
+
+* `myTask1` should include an `export` parameter with a list of these variable names,
+* `myTask2` should include an `import` parameter with a list including the same names.
+
+Example:
+
+`myTask1` implementation bloc would be:
+```python
+#%task(name=myTask1, export=[var1,var2])
+var1 = "Hello"
+var2 = "ActiveEon!"
+```
+
+and `myTask2` implementation bloc would be:
+```python
+#%task(name=myTask2, dep=[myTask1], import[var1,var2])
+print(var1 + " from " + var2)
+```
+
+===== Implementation file
+
+It is possible to use an external implementation file as a task implementation. To do so, the option `path` should be used.
+Example:
+
+```python
+#%task(name=myTask,path=PATH_TO/IMPLEMENTATION_FILE.py)
+```
+
+==== Importing libraries
+
+The main difference between ProActive kernel and a native language one resides in the memory access during the
+execution of the different blocs.  While in a common native language kernel the whole script code (all the notebook
+blocs) is executed locally in the same shared memory zone, the ProActive kernel each created task will be executed as
+an independent process.
+To facilitate the transition from native language kernels to the ProActive one we included the pragma `#%import()`.
+This pragma gives the ability to the user to add, just once in the notebook, libraries that are common to all created
+tasks implemented in a same native script language.
+
+The pragma is used in this general manner: `#%import([language=SCRIPT_LANGUAGE])`. Example:
+
+```python
+#%import(language=Python)
+import os
+import pandas
+```
+
+Notice that if the language is not specified, Python is considered by default.
+
+==== Adding a fork environment
+
+To configure a fork environment for a task, use the `#%fork_env()` pragma. A first way to do this
+is by providing the name of the corresponding task, and the fork environment implementation after that:
```

Comment:
-after that? what does mean?
-To do so, you can provide the task name, and ...

Reply:
And now?

To configure a fork environment for a task, use the `#%fork_env()` pragma. To do so, you have to provide the name of the corresponding task and the fork environment implementation: